File: TransportManagment.API/TransportManagmentInfrustructure/Model/Vehicle/Configuration/ServiceType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TransportManagmentInfrustructure.Data;
using TransportManagmentInfrustructure.Model.Authentication;
using static TransportManagmentInfrustructure.Enums.CommonEnum;

namespace TransportManagmentInfrustructure.Model.Vehicle.Configuration
{
    public class ServiceType: SettingIdModel
    {
        [StringLength(ValidationClasses.MaxNameLength)]
        [Required]
        public string Name { get; set; } = null!;
        [StringLength(ValidationClasses.MaxLocalNameLength)]
        [Required]
        public string LocalName { get; set; } = null!;
        public ServiceModule ServiceModule { get; set; }
        [StringLength(ValidationClasses.MaxSettingRemarkLength)]
        [Required]
        public string ListOfPlates { get; set; } = null!;
        [StringLength(ValidationClasses.MaxSettingRemarkLength)]
        [Required]
        public string ListOfAIS { get; set; } = null!;

    }
}

File: TransportManagment.UI/src/app/model/vehicle-configuration/service-type.ts
export interface ServiceTypeGetDto {
    id: number;
    name: string;
    localName: string;
    serviceModule: string;
    listOfPlates: string;
    listOfAIS: string;
    createdById: string;
    isActive:Boolean;
    
}
export interface ServiceTypePostDto {
    name: string;
    localName: string;
    serviceModule: string;
    listOfPlates: string;
    listOfAIS: string;
    createdById: string;
    isActive:Boolean;
}